modules/atomic-widgets/widgets/atomic-heading.php

```diff
@@ -104,8 +104,8 @@ protected function define_atomic_controls(): array {
 
 					Textarea_Control::bind_to( 'title' )
 						->set_label( __( 'Title', 'elementor' ) )
-						->set_placeholder( __( 'Type your title here', 'elementor' ) ),
-
+						->set_placeholder( __( 'Type your title here', 'elementor' ) )
+						->set_value( '' ),
```

Comment:
If we add a fallback in the function itself than it can be removed from here 
```suggestion
						->set_value(),
```